recipes/botan/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,13 @@
+project(test_package)
+cmake_minimum_required(VERSION 2.8.11)
```

```suggestion
project(test_package)
```
`cmake_minimum_required` needs to be at the top of the file